scripts/extract-translations.js

```diff
@@ -1,14 +1,15 @@
 #!/usr/bin/env node
 /* eslint-disable  */
 /* disable-translation */
+/* disable-old-translation */
 const path = require('path')
 const program = require('commander')
 const crc32 = require('crc-32').str
 const fs = require('fs')
 const glob = require('glob')
 const translated_keys = require('../src/translations/ach.json')
 const DISABLE_TRANSLATION = 'disable-translation'
-
+const DISABLE_OLD_TRANSLATION = 'disable-old-translation'
```

Comment:
```suggestion
```
@niloofar-deriv As discussed, We need to update the `old_find_keys` method to skip lines matching `new_i18n_marker`, So there won’t be a need for any flags.